tests/main.cpp

```diff
@@ -33,9 +33,9 @@ TEST_CASE("Simulate & Send Transaction") {
 
   const auto simulateRes = connection.simulateTransaction(keyPair, compiledTx);
   // consumed units should be greater than unity
-  CHECK_GT(simulateRes["unitsConsumed"], 0);
+  CHECK_GT(simulateRes.unitsconsumed, 0);
```

Comment:
Capitalization should stay the same so it’s easy to search for api methods / fields